tests/unit/plugins/modules/test_nmcli.py

```diff
@@ -4485,8 +4485,10 @@ def test_bond_connection_unchanged(mocked_generic_connection_diff_check, capfd):
             macvlan=dict(type='dict'),
             wireguard=dict(type='dict'),
             vpn=dict(type='dict'),
-            transport_mode=dict(type='str', choices=['datagram', 'connected']),
             sriov=dict(type='dict'),
+            # infiniband specific vars
+            transport_mode=dict(type='str', choices=['datagram', 'connected']),
+            infiniband_mac=dict(type='str'),
```

Comment:
Why is there this entire new module definition here in the tests? Please don't tell me we test _this_ instead of the actual module... 😄

Reply:
The tests have been doing this for a longer time, and yes, it should be cleaned up :)

Reply:
:facepalm: